### ISSUE TITLE
Cannot find x-goog-api-key in Request Header of issue_jwt

### ISSUE THREAD

**Issue by ssllii** [chrisjshull/homebridge-nest#338]
**Describe the bug**
Unable to config for Google account

**To Reproduce**
1. Followed the instructions for Google Account.  Unable to find x-goog-api-key in Request Header of last issue_jwt to finish the config.

**Expected behavior**

Include with your bug report this version info:
```sh
node --version
homebridge --version
```

Make sure you have the latest LTS from https://nodejs.org
and the latest packages: `npm upgrade -g homebridge homebridge-nest`

Also include debug log output from startup through seeing the issue:
`DEBUG=* homebridge -D`

**Comment by adriancable** [comment]
@ssllii - can you post a screenshot of **the whole DevTools window** showing the request headers of this call? Thanks!

**Comment by ssllii** [comment]
![2020-11-24-152025_1920x1080_scrot](https://user-images.githubusercontent.com/29691593/100169441-61669c80-2e78-11eb-85b2-d5dc5f85a913.png)

I logged out of my google accounts, signed out from Chromium, created a new tab, clear cookies and did the home.nest.com access from a new tab.  The attached is a screenshot.  
thanks for the quick response.

**Comment by adriancable** [comment]
@ssllii - as an experiment can you just set apiKey to blank in the config.json? i.e.

`"apiKey": ""`

**Comment by ssllii** [comment]
Ok, I finally got x-goog-api-key. The only thing I really changed was to un-configure one plug-in that was given me some other problems. 

Now I got the homebridge-nest configured. Part 1 done. 

However, both my Thermostats are showing up as security devices and no way to tell which one is which? In my Google Home, I have given the thermostat different Device Name.  Did I miss something on the config?
